### DISCUSSION TITLE
I found a small UI bug in GitHub!

### DISCUSSION THREAD

**Discussion by vashukashyap** [community/community#67714] [Other Feature Feedback, Questions, & Ideas]
# I found a small UI bug in GitHub interface.

## Step to reproduce

- Open your GitHub account.
- Go to dashboard.
-  In the left pane, Hover cursor any of the link.
- A small box will appear telling about the information of the link.
- now scroll.

The small box will also start moving with the screen

    

## While Hovering
![git issue](https://github.com/community/community/assets/129667408/687d6e49-f678-4e63-9161-d010ece5aae8)

## While Scrolling
![git_issue](https://github.com/community/community/assets/129667408/efca4d61-b0a4-43f6-9e36-667442994b3b)

----

#### but I don't know how to tell it to GitHub? and I also want to know, can I work on that issue?

**Comment by RumeysaaKopuz** [comment]
response #2
